_posts/2020-05-26-exploring-the-open-sgid-a-brief-tutorial-using-open-source-python-tools.md

```diff
@@ -0,0 +1,146 @@
+
+```py
+# Davis County address points by zip code
+sql_addpts = "select * from opensgid.location.address_points where countyid = '49011'"
+addpts = gpd.GeoDataFrame.from_postgis(sql_addpts, con, geom_col='shape')
+
+sql_davis = "select * from opensgid.boundaries.county_boundaries where name = 'DAVIS'"
+davis = gpd.GeoDataFrame.from_postgis(sql_davis, con, geom_col='shape')
+
+# Plot address points symbolized by zip code
+fig, ax = plt.subplots(figsize=(10, 6))
+davis.plot(ax=ax, color='lightgray', edgecolor='black')
+addpts.plot(ax=ax, column='zipcode', cmap='jet', markersize=1, legend=True,
+            legend_kwds={'title': 'Zip Code',
+                         'loc': 'lower left'})
+plt.title('Davis County Address Points')
+plt.show()
+```
+
+[![Davis County Address Points]({% link images/EMN_20200526_addpts_plot.png %} "Davis County Address Points")]({% link images/EMN_20200526_addpts_plot.png %}){: .flex .flex--center }
+
+### Utah Faults Longer than One Mile
+
+In the third example, we'll demonstrate the subsetting capabilities of Pandas/GeoPandas to find and plot fault lines in Utah that are longer than 1 mile (1609.34 meters).  First, we'll specify the layer we want to work with in our SQL query.  Second, we'll create a subset of all the faults (called 'long_faults') by selecting those where the 'shape' field has a length greater than 1609.34 meters.  We use `.copy()` at the end of this line of code to prevent a 'chained assignment' warning in Pandas.  Next, we'll calculate a new field in this subset that converts the length into miles, for easier interpretation.  Finally, we plot the counties and faults longer than 1 mile, colored by length.  Note that the syntax for the legend is a little different now that we are plotting a continuous variable (lengths a floating point) rather than a discrete variable (zipcodes as strings).  We also specify the starting and ending ranges for the colorbar (`vmin` and `vmax`).
```

Comment:
Can't decide the correct syntax to achieve your desired outcome, but something like 
(lengths as floating points) OR 
(length as floating point) OR 
(length is a floating point), etc.

Reply:
Yeah, we've cleaned this up in the text revisions I'm working with Anna.